### PR TITLE
fix go-statickcheck complaints

### PR DESCRIPTION
**Pull request by hj-johannes-lee**
I do not know whether I can make a pull request on this part that I didn't write or not, but I just wanted to help in some way.!

---
fix go-statickcheck complaints

fix capitalized letters at the beginning of errors to have small letters
fix corev1 to v1 that is already imported

### CODE REVIEW

**Comment by pohly** [comment]
I can take that last commit at some point, but right now I am more interested in getting the example driver updated.

File: test/integration/cdi/example-driver/app/controller.go [hunk 1]
```diff
@@ -50,13 +49,13 @@ type parameters map[string]string
 
 var _ controller.Driver = exampleDriver{}
 
-func (d exampleDriver) GetClassParameters(ctx context.Context, class *corev1.ResourceClass) (interface{}, error) {
+func (d exampleDriver) GetClassParameters(ctx context.Context, class *v1.ResourceClass) (interface{}, error) {
```
Comment (bart0sh) [inline test/integration/cdi/example-driver/app/controller.go:52]: 'corev1' as more informative and specific than 'v1' in my opinion.

**Comment by bart0sh** [comment]
@hj-johannes-lee please, rebase

**Comment by hj-johannes-lee** [comment]
@bart0sh I guess Patrick already fixed these in some other commits. I just close this.

**Pull request closed without merging**